edu.gemini.lch.services/src/main/scala/edu/gemini/lch/services/impl/JSpOCCredentials.scala

```diff
@@ -0,0 +1,19 @@
+package edu.gemini.lch.services.impl
+
+import edu.gemini.lch.model.Site
+import org.apache.commons.httpclient.UsernamePasswordCredentials
+
+import scala.collection.immutable.HashMap
+
+object JSpOCCredentials {
```

Comment:
Where is this used?

Reply:
https://github.com/gemini-hlsw/lch/blob/dd2f60b74392927b5c7322913ba466b18ac199b6/edu.gemini.lch.services/src/main/scala/edu/gemini/lch/services/impl/JSpOCCommunicatorServiceImpl.scala#L49

Reply:
It's used to log into space-track to transfer files.

Reply:
Ok thanks, I was confused because it seemed to be a new file but I suppose that's just because before it was kept in svn and linked in.

Reply:
Yes we did the link trick. But it is hard to do on github actions without revealing a pwd for svn